src/control/server/ctl_smd_rpc.go

```diff
@@ -340,6 +349,33 @@ func addManageRespIDOnFail(log logging.Logger, res *ctlpb.SmdManageResp_Result,
 	}
 }
 
+// Retry dev-replace requests as state propagation may take some time after set-faulty call has
+// been made to manually trigger a faulty device state.
+func replaceDevRetryBusy(ctx context.Context, log logging.Logger, e Engine, req proto.Message) (res *ctlpb.SmdManageResp_Result, err error) {
+	for try := uint(0); try < uint(maxDevReplaceRetries); try++ {
```

Comment:
nit - don't need to cast to uint -- these are both regular ints.

Reply:
correct I forgot to change this back after correcting elsewhere, I will change in a follow-up, thanks